### PR TITLE
CORCI-1027 test: Blacklist utils/rpms from codespell and fix misspellings.

### PR DESCRIPTION
**Pull request by ashleypittman**
Signed-off-by: Ashley Pittman <ashley.m.pittman@intel.com>

### CODE REVIEW

File: ci/patch_src_in_place [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # shellcheck disable=SC2046,SC2035
-codespell -w --ignore-words-list dedup,nd,uint,ths,ba,creat,te,cas,mapp,pres,crashers,dout,tre,reord,mimick,cloneable,keypair,bject,tread,cancelled,controllerd --builtin clear,rare,informal,names,en-GB_to_en-US --skip *.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden $(git ls-tree --full-tree --name-only HEAD)
+codespell -w --ignore-words-list dedup,nd,uint,ths,ba,creat,te,cas,mapp,pres,crashers,dout,tre,reord,mimick,cloneable,keypair,bject,tread,cancelled,controllerd,gae,fpr,convience --builtin clear,rare,informal,names,en-GB_to_en-US --skip *.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden $(git ls-tree --full-tree --name-only HEAD)
```
Comment (mjmac) [inline ci/patch_src_in_place:4]: Should we ignore `convience `? Seems like a tyop...

Comment (mjmac) [reply ci/patch_src_in_place:4]: Ah, hmm. But you're skipping `src/control/vendor/*`, right?

Comment (ashleypittman) [reply ci/patch_src_in_place:4]: Good point.  I can't find a build which has failed now, if you can point me at one I'll check.

**Comment by ashleypittman** [comment]
This doesn't seem to be a problem any more, not sure why but perhaps someone else fixed it.  I'll park this PR for now.

**Review by jolivier23 — APPROVED** [review]

**Review by marcelarosalesj — DISMISSED** [review]

File: ci/patch_src_in_place [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # shellcheck disable=SC2046,SC2035
-codespell -w --ignore-words-list dedup,nd,uint,ths,ba,creat,te,cas,mapp,pres,crashers,dout,tre,reord,mimick,cloneable,keypair,bject,tread,cancelled,controllerd --builtin clear,rare,informal,names,en-GB_to_en-US --skip *.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden $(git ls-tree --full-tree --name-only HEAD)
+codespell -w --ignore-words-list dedup,nd,uint,ths,ba,creat,te,cas,mapp,pres,crashers,dout,tre,reord,mimick,cloneable,keypair,bject,tread,cancelled,controllerd --builtin clear,rare,informal,names,en-GB_to_en-US --skip *.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden,utils/rpm/_topdir/* $(git ls-tree --full-tree --name-only HEAD)
```
Comment (ashleypittman) [inline ci/patch_src_in_place:4]:
```suggestion
codespell -w --ignore-words-list dedup,nd,uint,ths,ba,creat,te,cas,mapp,pres,crashers,dout,tre,reord,mimick,cloneable,keypair,bject,tread,cancelled,controllerd --builtin clear,rare,informal,names,en-GB_to_en-US --skip *.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden,utils/rpms/* $(git ls-tree --full-tree --name-only HEAD)
```

**Review by mjmac — DISMISSED** [review]

**Review by mjmac — DISMISSED** [review]

File: ci/patch_src_in_place (hunk 2 above)
Comment (ashleypittman) [inline ci/patch_src_in_place:4]:
```suggestion
codespell -w --ignore-words-list dedup,nd,uint,ths,ba,creat,te,cas,mapp,pres,crashers,dout,tre,reord,mimick,cloneable,keypair,bject,tread,cancelled,controllerd --builtin clear,rare,informal,names,en-GB_to_en-US --skip *.png,*.PNG,*.pyc,src/rdb/raft/*,src/control/vendor/*,RSA.golden,utils/rpms/* $(git ls-tree --full-tree --name-only HEAD)
```

**Review by brianjmurrell — APPROVED** [review]

**Comment by ashleypittman** [comment]
I checked the output this time and it's correct, although this PR isn't actually affected by the bug so we can't fully check it until this is testing with a PR that is.

**Review by tanabarr — APPROVED** [review]

**Pull request merged**